app/src/main/java/io/hobaskos/event/eventapp/ui/base/BaseMvpFragment.java

```diff
@@ -15,7 +15,7 @@
         extends Fragment {
 
     private static final String TAG = "base-fragment";
-    private static final int LOADER_ID = 101;
+    private static final int LOADER_ID = 102;
```

Comment:
Slik jeg forstår det så skal IDen være unik for aktiviteten/fragmentet. Det er først hvis vi ønsker enda en loader til fragmenter som f.eks gjør noe annet at den trenger en annen id. 
"The LoaderManager manages one or more Loader instances within an Activity or Fragment. There is only one LoaderManager per activity or fragment."